Update Presentation API samples

beaufortfrancois opened the pull request:
I've spent some time updating our Presentation API samples to make them easier to read.
WDYT @mfoltzgoogle?

Live previews:
https://beaufortfrancois.github.io/samples/presentation-api/
https://beaufortfrancois.github.io/samples/presentation-api/cast.html

beaufortfrancois committed:
Update Presentation API samples

markafoltz commented on presentation-api/receiver/receiver.js:
```diff
@@ -1,65 +1,62 @@
+    const logString = 'Message ' + messageIdx + ' from connection #' +
+        connection.connectionId + ': ' + data.message;
+    addMessage(logString, data.lang);
+    const fruit = data.message.toLowerCase();
```
Might help to factor out a maybeSetFruit(data.message) function here.

beaufortfrancois replied:
Done.

markafoltz commented on presentation-api/cast.html:
```diff
@@ -17,97 +17,86 @@ <h3>Background</h3>
-var presentationUrl = 'cast:EF1A139F';
-var presentationRequest;
-var presentationConnection;
+const presentationRequest = new PresentationRequest('cast:EF1A139F');
```
Can you set navigator.presentation.defaultRequest here, so this presentation is the default choice when casting from the toolbar/menu?

beaufortfrancois replied:
Done.

markafoltz commented on presentation-api/index.html:
```diff
@@ -41,122 +37,97 @@ <h3>Background</h3>
+
 {% capture js %}
-const presentationUrls = ['receiver/index.html'];
+const presentationRequest = new PresentationRequest(['receiver/index.html']);
```
Can you set navigator.presentation.defaultRequest here, so this presentation is the default choice when casting from the toolbar/menu?

beaufortfrancois replied:
Done.

markafoltz commented on presentation-api/cast.html:
```diff
@@ -17,97 +17,86 @@ <h3>Background</h3>
+document.querySelector('#reconnect').addEventListener('click', () => {
+  const presentationId = document.querySelector('#presentationId').value.trim();
+
+  presentationRequest.reconnect(presentationId)
```
reconnect() may not work for Cast URLs with the presentationConnection.id as given...

beaufortfrancois replied:
I've removed it then. Thanks!

markafoltz reviewed (commented):
Thanks for the improvements! This looks nicer.
A few comments but fine if you would like me to merge now.

beaufortfrancois committed:
Address @mfoltzgoogle feedback

beaufortfrancois commented:
@mfoltzgoogle I've addressed your feedback at https://github.com/GoogleChrome/samples/pull/564/commits/99e2b3fe78e16dc6c46befa74d8e65bad9f4e9df.

Updated live previews:
https://beaufortfrancois.github.io/samples/presentation-api/
https://beaufortfrancois.github.io/samples/presentation-api/cast.html

markafoltz reviewed (approved):
LGTM!  Thanks!
